athenahealth/generic.go

```diff
@@ -9,3 +15,20 @@ type ErrorMessageResponse struct {
 	Message string `json:"errormessage"`
 	Success bool   `json:"success"`
 }
+
+// BodyToErrorString will handle the obtuse case where Athena returns an error
+// message as a raw string with zero JSON formatting.
+func BodyToErrorString(body io.ReadCloser) error {
```

Comment:
Can then remove this, as this functionality is covered by the NumberString & req logic